src/main/java/seedu/address/model/person/Address.java

```diff
@@ -25,9 +28,12 @@
      * @param address A valid address.
      */
     public Address(String address) {
-        requireNonNull(address);
-        checkArgument(isValidAddress(address), MESSAGE_CONSTRAINTS);
-        value = address;
+        if (address == null) {
+            value = null;
+        } else {
```

Comment:
Can consider cleaning this up into short lines for better readability, but not a big deal!